cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -12,6 +12,81 @@ WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
 
 OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 
+function ensure_pod {
+  namespace=$1
+  label=$2
+
+  intervals=6
+  timeout=10
+
+  if [[ $namespace != "" ]];then
+    namespace="-n $namespace"
+  fi
+
+  if [[ $label != "" ]];then
+    label="-l $label"
+  fi
+
+  count=0
+  until _kubectl get pod $namespace $label; do
+      ((count++)) && ((count == intervals)) && echo "Pods with $label not found" && exit 1
+      echo "[$count/$intervals] Waiting for pods with $label to create.."
+      sleep $timeout
+  done
+}
+
+function ensure_k8s_object {
+  object_type=$1
+  name=$2
+  namespace=$3
+
+  intervals=30
+  timeout=3
+
+  if [[ $namespace != "" ]];then
+    namespace="-n $namespace"
+  fi
+
+  count=0
+  until _kubectl get $object_type $name $namespace; do
+      ((count++)) && ((count == intervals)) && echo "$object_type $name not found" && exit 1
+      echo "[$count/$intervals] Waiting for $object_type $name.."
+      sleep $timeout
+  done
+}
+
+function wait_for_taint {
+  taint=$1
+
+  intervals=30
+  timeout=10
+  let "total = ($timeout * $intervals) / 60"
+
+  count=0
+  until [[ $(_kubectl  get nodes -ocustom-columns=taints:.spec.taints[*].effect --no-headers | grep -i $taint) != "" ]]; do
```

Comment:
this and line 82 are almost the same, beside the exit and break,
maybe worth a function ?

also there is a double space before `get nodes`

Reply:
done